web/application.py

```diff
@@ -410,9 +407,7 @@ def gaerun(self, *middleware):
             minor = version[1]
 
             if major != 2:
-                raise EnvironmentError(
-                    "Google App Engine only supports python 2.5 and 2.7"
-                )
+                raise OSError("Google App Engine only supports python 2.5 and 2.7")
```

Comment:
Question: Google App Engine (GAE) supports Python 2 + 3, should we improve webpy to support GAE with Python3, or we drop GAE completely?

Reply:
Let's do that in a separate PR.